vector/src/main/scala/geotrellis/vector/Projected.scala

```diff
@@ -3,8 +3,11 @@ package geotrellis.vector
 import geotrellis.proj4.CRS
 
 object Projected {
-  implicit def fromTupleA[T](tup: (T, CRS)):ProjectedExtent = Projected(tup._1, tup._2)
-  implicit def fromTupleB[T](tup: (CRS, T)):ProjectedExtent = Projected(tup._2, tup._1)
+  implicit def fromTupleA[T](tup: (T, CRS)): Projected[T] = Projected(tup._1, tup._2)
+  implicit def fromTupleB[T](tup: (CRS, T)): Projected[T] = Projected(tup._2, tup._1)
 }
 
-case class Projected[T](obj: T, crs: CRS) extends Product2[T, CRS]
\ No newline at end of file
+case class Projected[T](obj: T, crs: CRS) extends Product2[T, CRS] {
```

Comment:
@echeipesh there is an important fix, think we can rebase other pr's on top of this / make this small fix as a separate pr.